sos/report/plugins/openstack_cinder.py

```diff
@@ -48,6 +49,72 @@ def setup(self):
                 "cinder-manage " + cinder_config + " db version",
                 suggest_filename="cinder_db_version"
             )
+            self.add_cmd_output(
+                f"cinder-manage {cinder_config} backup list",
+                suggest_filename="cinder-manage_backup_list"
+            )
+            self.add_cmd_output(
+                f"cinder-manage {cinder_config} config list",
+                suggest_filename="cinder-manage_config_list"
+            )
+            self.add_cmd_output(
+                f"cinder-manage {cinder_config} host list",
+                suggest_filename="cinder-manage_host_list"
+            )
+            self.add_cmd_output(
+                f"cinder-status {cinder_config} upgrade check",
+                suggest_filename="cinder-status_upgrade_check"
+            )
+
+            vars_all = [p in os.environ for p in [
+                        'OS_USERNAME', 'OS_PASSWORD']]
+
+            vars_any = [p in os.environ for p in [
+                        'OS_TENANT_NAME', 'OS_PROJECT_NAME']]
+
+            if not (all(vars_all) and any(vars_any)):
+                self.soslog.warning("Not all environment variables set. "
+                                    "Source the environment file for the user "
+                                    "intended to connect to the OpenStack "
+                                    "environment.")
```

Comment:
Maybe add an information this execution is skipping some commands execution? Either explicitly, or e.g. by appending something like ".. to get further data collected".

But I am OK with the current text, it does contain an explicit request.